src/lavague/defaults.py

```diff
@@ -77,3 +76,16 @@ def default_get_driver() -> SeleniumDriver:
 
 def defaultTestCode() -> str:
     return 'driver.execute_script("window.scrollBy(0, 500)")'
+
+
+def default_get_playwright_driver() -> PlaywrightDriver:
+    try:
+        from playwright.sync_api import sync_playwright
+    except (ImportError, ModuleNotFoundError) as error:
+        raise ImportError(
+            "Please install playwright using `pip install playwright` and then `playwright install` to install the necessary browser drivers"
+        ) from error
+    p = sync_playwright().__enter__()
+    browser = p.chromium.launch()
+    page = browser.new_page()
+    return PlaywrightDriver(page)
```

Comment:
Why have we moved to sync api ?? Are you able to run the buld command locally with sync API. It should not. So I used async API in my PR.

Reply:
build command is working with sync,
but jupyter notebooks are not working with the generated code, as you said.

Now, the problem with async api, is that it introduce a lof of async/await function in the code base, without real reason (because we're not running a server that needs multiple users or something).

Also the generated code looks ugly with async API.